mysql-test/suite/binlog/t/binlog_mysqlbinlog_warn_stop_position.test

```diff
@@ -0,0 +1,20 @@
+--echo
+--echo # MDEV-27037 mysqlbinlog emits a warning when reaching EOF before stop-condition
+--echo
+
+--let assert_file= $MYSQLTEST_VARDIR/tmp/warn_pos_test_file.out
+--let data_file= $MYSQLTEST_VARDIR/std_data/master-bin.000001
```

Comment:
Though another thought comes to me now. In `10.11`, `--stop-position` also supports a GTID value, and this test binlog file doesn't have any transactions. We should have a `10.11` specific patch to also warn when a GTID specified is not hit. And for that we will need transactions. It might be easier if we prepare for that here, and revert  back to something like your last test file which used its own binlog.

Reply:
Great idea, I didn't know about that additional support in 10.11. 👍

Reply:
I created https://jira.mariadb.org/browse/MDEV-34614 to capture this work as it's in a different version of the codebase.